crates/swc/tests/libs-size.snapshot.md

```diff
@@ -1,14 +1,14 @@
 | File | Original Size | Compressed Size | Gzipped Size |
 | --- | --- | --- | --- |
-| antd.js | 6.38 MiB | 2.05 MiB | 444.47 KiB |
-| d3.js | 542.74 KiB | 259.00 KiB | 85.06 KiB |
-| echarts.js | 3.41 MiB | 971.11 KiB | 313.35 KiB |
-| jquery.js | 280.89 KiB | 87.11 KiB | 30.15 KiB |
-| lodash.js | 531.35 KiB | 68.18 KiB | 24.52 KiB |
-| moment.js | 169.83 KiB | 57.04 KiB | 18.22 KiB |
+| antd.js | 6.38 MiB | 2.05 MiB | 444.51 KiB |
+| d3.js | 542.74 KiB | 259.00 KiB | 85.05 KiB |
+| echarts.js | 3.41 MiB | 971.29 KiB | 313.36 KiB |
+| jquery.js | 280.89 KiB | 87.08 KiB | 30.14 KiB |
+| lodash.js | 531.35 KiB | 68.18 KiB | 24.51 KiB |
+| moment.js | 169.83 KiB | 57.05 KiB | 18.22 KiB |
 | react.js | 70.45 KiB | 22.28 KiB | 8.00 KiB |
-| terser.js | 1.08 MiB | 444.91 KiB | 120.44 KiB |
-| three.js | 1.19 MiB | 627.90 KiB | 154.58 KiB |
-| typescript.js | 10.45 MiB | 3.17 MiB | 846.04 KiB |
-| victory.js | 2.30 MiB | 689.65 KiB | 153.64 KiB |
-| vue.js | 334.13 KiB | 112.96 KiB | 41.69 KiB |
+| terser.js | 1.08 MiB | 444.95 KiB | 120.45 KiB |
+| three.js | 1.19 MiB | 627.99 KiB | 154.58 KiB |
+| typescript.js | 10.45 MiB | 3.17 MiB | 846.08 KiB |
+| victory.js | 2.30 MiB | 689.66 KiB | 153.64 KiB |
+| vue.js | 334.13 KiB | 112.98 KiB | 41.69 KiB |
```

Comment:
Why there're two size test and their results are different

Reply:
I think the difference comes from codegen option difference. It would be fine to remove the one in `swc_ecma_minifier` IMO

Reply:
I'll remove one in `swc` with another PR after merging this